src/main/resources/library/attribute-definitions.json

```diff
@@ -334,14 +334,14 @@
     "library:lmsvn": {
       "title": "Library Metadata Schema Version Number",
       "description": "Library Metadata Schema Version Number",
-      "default": 5,
+      "default": 6,
```

Comment:
@davidangb @ahaessly - I bumped these versions up again because we've already bumped them on dev. However, we only need one version bump for prod. I'm open to reverting these if if makes sense - let me know.